dex/plan.sh

```diff
@@ -7,31 +7,39 @@ pkg_maintainer="Chef Software Inc. <support@chef.io>"
 pkg_license=("Apache-2.0")
 pkg_source="https://$gopkg"
 pkg_upstream_url=$pkg_source
-pkg_build_deps=()
 pkg_exports=(
   [port]=service.port
   [host]=service.host
 )
-pkg_deps=()
+pkg_deps=(core/glibc)
```

Comment:
Do we know that this won't run (our tests) without that? Just wondering if it was carried over from scaffolding-go or put there with intent.

Reply:
Yup, we need glibc in our deps:  

```
ldd /hab/pkgs/smacfarlane/dex/2.13.0/20181217161258/bin/dex
linux-vdso.so.1 (0x00007fff8a96e000)
libdl.so.2 => /hab/pkgs/core/glibc/2.27/20180608041157/lib/libdl.so.2 (0x00007f5032ba6000)
libpthread.so.0 => /hab/pkgs/core/glibc/2.27/20180608041157/lib/libpthread.so.0 (0x00007f5032988000)
libc.so.6 => /hab/pkgs/core/glibc/2.27/20180608041157/lib/libc.so.6 (0x00007f50325d4000)
/hab/pkgs/core/glibc/2.27/20180608041157/lib/ld-linux-x86-64.so.2 => /hab/pkgs/core/glibc/2.27/20180608041157/lib64/ld-linux-x86-64.so.2 (0x00007f5032daa000)
```